### PR TITLE
COLDBOX-567

### PR DESCRIPTION
**Pull request by bdw429s**

### CODE REVIEW

**Comment by bdw429s** [comment]
https://ortussolutions.atlassian.net/browse/COLDBOX-567

**Comment by bdw429s** [comment]
@lmajano

File: system/testing/BaseTestCase.cfc [hunk 1]
```diff
@@ -41,7 +41,8 @@ component extends="testbox.system.compat.framework.TestCase"  accessors="true"{
 	* @return BaseTestCase
 	*/
 	function metadataInspection(){
-		var md = getMetadata( this );
+		var md = new coldbox.system.core.util.util().getInheritedMetadata( this );
```
Comment (lmajano) [inline system/testing/BaseTestCase.cfc]: This won't work in case sensitive systems. The file is Útil not util

Comment (bdw429s) [inline system/testing/BaseTestCase.cfc]: Updated file name

**Comment by bdw429s** [comment]
@lmajano We need to update any Travis builds that are copying XML config files on start to meet the CommandBox 3.5 folder structure for servers.
```
Error executing command #fileCopy build/lib/lucee-server.xml /home/travis/.CommandBox/engine/cfml/server/lucee-server/context
```

**Comment by lmajano** [comment]
Can you tell me what is the new structure please? @bdw429s

**Comment by bdw429s** [comment]
If only this was thoroughly documented somewhere! ;)  
https://ortus.gitbooks.io/commandbox-documentation/content/embedded_server/copy-configs.html

**Comment by bdw429s** [comment]
@lmajano Do you want me to fix it as part of this pull?  Then we'll know if it works without having to rebase it into this branch?

**Comment by lmajano** [comment]
Go for it

Luis Majano
CEO
Ortus Solutions, Corp
www.ortussolutions.com
P/F: 1-888-557-8057




On Tue, Feb 7, 2017 at 4:38 PM +0100, "Brad Wood" <notifications@github.com> wrote:












@lmajano Do you want me to fix it as part of this pull?  Then we'll know if it works without having to rebase it into this branch?

**Comment by bdw429s** [comment]
@lmajano The build is passing now and even runs on Lucee 5 since it is compat with the Lucee 4 config.  When I get CFConfig finalized, we'll be able to add in Adobe easily as well.

File: box.json [hunk 2]
```diff
@@ -63,5 +63,8 @@
     },
     "installPaths":{
         "testbox":"testbox"
-    }
+    },
+	"scripts":{
+		onServerInstall: "cp build/lib/lucee-server.xml '`server info property=serverHomeDirectory`/WEB-INF/lucee-server/context/lucee-server.xml'"
```
Comment (lmajano) [inline box.json]: this in invalid JSON, add `"onServerInstall"`

**Comment by lmajano** [comment]
Merged manually

**Pull request closed without merging**